frameworks/cassandra/universe/config.json

```diff
@@ -129,22 +129,45 @@
                             "password_secret_path": {
                                 "type": "string",
                                 "description": "The path in the DC/OS Secret Store to retrieve the superuser password from" ,
-                                "default": "cassandra/password"
```

Comment:
You have to add this back as the default. Based on the marathon mustache if we enable `authentication` without any other changes, it will default to `PasswordAuthenticator` and [this line](https://github.com/mesosphere/dcos-commons/pull/3030/files#diff-fe0c20da1fba698643bc101afa99c204R286) will not export a missing file

Reply:
Correct, the reason i have removed is that when you trying for custom authentication you will not provide password in secret store, and  [secret](https://github.com/mesosphere/dcos-commons/pull/3030/files#diff-fe0c20da1fba698643bc101afa99c204L15) will fail to fetch.

Reply:
as discussed this would be the cleanest way to implement our changes.